src/MSSQLSchemaManager.php

```diff
@@ -135,27 +134,6 @@ public function getPrimaryKey($tableName)
         return $indexName;
     }
 
-    /**
-     * Gets the identity column of a table
-     *
-     * @param string $tableName
-     * @return string|null
-     */
-    public function getIdentityColumn($tableName)
```

Comment:
Why has this function been removed?

Reply:
As far as I could see it was no longer used anywhere?